scanner.cs

```diff
@@ -27,6 +27,7 @@ public class Scanner {
 
                 // Loop until EOF, ignoring whitespace
                 while(__curByte < length){
+                    // TODO: Handle comments like we handle whitespace
```

Comment:
We will need to handle comments, but those aren't given to the parser, we just skip over them